js/src/row/map.ts

```diff
@@ -107,7 +107,7 @@ class MapRowProxyHandler<K extends DataType = any, V extends DataType = any> imp
     deleteProperty() { return false; }
     preventExtensions() { return true; }
     ownKeys(row: MapRow<K, V>) {
-        return row[kKeys].toArray().map(String);
+        return Array.from(row[kKeys].toArray(), String);
     }
     has(row: MapRow<K, V>, key: string | symbol) {
         return row[kKeys].includes(key);
```

Comment:
(I’m not fixing this in this PR, but I suspect that there’s also a bug here: if `key` is a string, and `rows[kKeys]` is a typed array, then `includes(key)` will always return false. You’d need to coerce the `key` to a number, or coerce `rows[kKeys]` to strings, for this test to return true. Same with other usage of `row[kKeys].indexOf(key)` below, I expect.)

Reply:
@mbostock any reason not to fix it in this PR?

Reply:
I’m not familiar enough with this codebase to do it confidently and to test. I’d welcome someone else doing it.